torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11396,28 +11396,40 @@ def __call__(self, opinfo, device, dtype, requires_grad, **kwargs):
                 unittest.expectedFailure,
                 "TestMeta",
                 "test_dispatch_meta_inplace",
-                dtypes=integral_types_and(torch.bool),
+                dtypes=all_types_and(torch.bool, torch.bfloat16, torch.float16),
```

Comment:
This in effect cancels running the tests at all, right? Why are these now failing?

Reply:
Because ScalarList samples would cover more various dtypes in the list. I expect this to be better with #134552

Reply:
So this should be undone once this PR is rebased on that one?